solidity/hardhat.config.ts

```diff
@@ -111,10 +111,18 @@ const config: HardhatUserConfig = {
     goerli: {
       url: process.env.CHAIN_API_URL || "",
       chainId: 5,
-      accounts: process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY
-        ? [process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY]
+      accounts: process.env.ACCOUNTS_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_PRIVATE_KEYS.split(",")
         : undefined,
-      tags: ["tenderly"],
+      tags: ["etherscan", "tenderly"],
+    },
+    mainnet: {
+      url: process.env.CHAIN_API_URL || "",
+      chainId: 1,
+      accounts: process.env.ACCOUNTS_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_PRIVATE_KEYS.split(",")
+        : undefined,
```

Comment:
For mainnet we expect to import only one private key.
Can we update this config to be consistent with other projects (see: https://github.com/keep-network/keep-core/blob/a0817e53bc6ce1c85013ee254d24b88cc78b881a/solidity/ecdsa/hardhat.config.ts#L123-L130).

```suggestion
      accounts: process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY
        ? [process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY]
        : undefined,
```

Reply:
Yeah, let's make it consistent though... I was hoping to do this change cross-project and either use `process.env.ACCOUNTS_PRIVATE_KEYS` or - if we want to emphasize this should be a single account - `process.env.ACCOUNT_PRIVATE_KEY`.

Reply:
4b4c5dfb8c3ac36340714dc52910c0a6e2471442